orttraining/pytorch_frontend_examples/nvidia-bert/Nvidia.dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020 NVIDIA CORPORATION. All rights reserved.
```

Comment:
Is this file needed at all given that we have added Onnxrt.dockerfile?

Reply:
It was intended to run the original NVIDIA flow. I think we have since decided we only want to focus on ORT flow. So we can remove it. And possibly other files not relevant to ORT flow.